src/Draupnir.ts

```diff
@@ -275,11 +275,10 @@ export class Draupnir implements Client {
     if (isError(managementRoomProtectResult)) {
       return managementRoomProtectResult;
     }
-    void Task(draupnir.startupComplete());
     return Ok(draupnir);
   }
 
-  private async startupComplete(): Promise<void> {
+  public async startupComplete(): Promise<void> {
```

Comment:
nit: should probably mention that this is only used in index.ts for showing the client has started syncing.

Reply:
Fixed